tests/foreman/cli/test_import.py

```diff
@@ -0,0 +1,45 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Host Collection CLI"""
+
+from ddt import ddt
+from robottelo.test import CLITestCase
+from robottelo.cli.imprt import Import as ImportObj
+
+
+@ddt
+class TestImport(CLITestCase):
+    """Import CLI tests."""
+
+    """All default tests pass no options to the imprt object
+    In such case methods download a default data set from URL
+    specified in robottelo.properties
+    """
+    def test_import_orgs_default(self):
+        """@test: Import 3 organizations from the
+        predefined source
+
+        @feature: Import Organizations
+
+        @assert: 3 Organizations are created
+
+        """
+
+        rc = ImportObj.organization()
```

Comment:
Sorry, I could have explained in the test directly - it's because that's how many organizations we have in our default dataset we download from `export_tar.url`.
This test is designed for a particular dataset as we don't have anything else so far (will be rewritten in future)

I'll add this piece of info to the test docstrings